waterloop-site/src/static/ste/icons/email.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="16" viewBox="0 0 24 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Could you clarify what you mean by 'clean it up'? I see the prettify option but otherwise I'm not too familiar with what the goal is for this. I can't optically see any noticeable quality differences. 

I've moved them to the requested folder.

Reply:
SVGOMG doesn't affect the image at all, it just cleans up the markup so that any unnecessary metadata is cleaned up - the file size gets smaller. It also formats it, which our Prettier configuration doesn't.

Reply:
Requested change completed.